src/sentry/api/bases/avatar.py

```diff
@@ -40,8 +56,12 @@ def get_avatar_filename(self, obj):
         return f"{obj.id}.png"
 
     def put(self, request, **kwargs):
-        obj = kwargs[self.object_type]
-        serializer = AvatarSerializer(data=request.data, context=self.get_serializer_context(obj))
+        obj = kwargs.pop(self.object_type, None)
+        serializer = (
+            SentryAppLogoSerializer(data=request.data, context=self.get_serializer_context(obj))
+            if self.object_type == "sentry_app"
+            else AvatarSerializer(data=request.data, context=self.get_serializer_context(obj))
```

Comment:
hmm, why do we need a customer serializer here for this, can't we just use the AvatarSerializer?

Reply:
I figured I'd just make it it's own because I wanted to limit the avatar types and validate the color attribute which doesn't exist elsewhere